tests/gabbits/syntaxes.yaml

```diff
@@ -12,3 +12,36 @@ tests:
         - lua
         - python
         - rust
+
+
+  - name: test /syntaxes (explicitly accept json)
+    GET: /v1/syntaxes
+    request_headers:
+      accept: "application/json"
+    status: 200
+    response_json_paths:
+      $:
+        - clojure
+        - json
+        - lua
+        - python
+        - rust
+
+  - name: test /syntaxes (explicitly accept any content type)
+    GET: /v1/syntaxes
+    request_headers:
+      accept: "*/*"
```

Comment:
What do you think about also testing negotiation with some 

```yaml
accept: "text/html; q=0.9, application/json"
```

and some 

```yaml
accept: "application/xml, application/json"
```

Reply:
Good point! Added a pair of such tests to every API route.